calculator.rb

```diff
@@ -0,0 +1,66 @@
+operand = ()
+# Method to get and validate operands
+def get_operand
+  operand = gets.chomp
+  while operand.empty? || (!operand.match("^[\-0-9]*$"))
+    puts "Invalid input. Please enter a number"
+    operand = gets.chomp
+  end
+  return operand.to_f
+end
+
+# Get user input for operator, set casing and convert to string.
+puts "Which one of the following operators would you like to use:
+
+Add (+) 
+Subtract(-) 
+Multiply(*)
+Divide(/)
+Remainder(%)
+Exponify(**)
+
+Please select an operator"
+operator = gets.chomp.to_s.downcase
```

Comment:
Because you mentioned that you would like to make more methods, and also because your `get_operand` method was really nice, I may suggest making a `get_operator` method. Even if a method to get the operator is used once, the benefits of the organization can be worth it!